index.html

```diff
@@ -36,102 +44,203 @@ <h3>LET'S CONNECT</h3>
+                        <ul class="languagess">
+                            <li class="languages" id="html1">HTML</li>
+                            <li class="languages" id="css1">CSS</li>
+                            <li class="languages" id="javascript1">Javascript</li>
                         </ul>
+                        <button class="buttons" type="button">see project</button>
                     </div>
-                    <p class="info">A daily selection of privately personalized reads; no accounts
-                        or sign-ups required.
-                    </p>
-                    <ul class="languages">
-                        <li class="language" id="html">HTML</li>
-                        <li class="language" id="css">CSS</li>
-                        <li class="language" id="javascript">Javascript</li>
-                    </ul>
-                    <button class="button">see project</button>
+                    <img class="snapshot" id="snap1" src="snap.svg">
                 </div>
-            </div>
-            <div class="projects">
-                <img class="snapshots" id="snap3" src="snap2.svg">
-                <div class="info_block">
-                    <div class="projo">
-                        <h4 class="project_name">Tonic</h4>
-                        <ul class="deatils">
-                            <li class="detail">CANOPY</li>
-                            <li class="detail"><img src="count.svg"></li>
-                            <li class="detail">Back End Dev</li>
-                            <li class="detail"><img src="count.svg"></li>
-                            <li class="detail">2023</li>
+                <div class="project">
+                    <img class="snapshot" id="snap2" src="snap1.svg">
+                    <div class="info_blocks">
+                        <div class="projos">
+                            <h4 class="project_names">Facebook 360</h4>
+                            <ul class="detailss">
+                                <li class="details">FACEBOOK</li>
+                                <li class="details"><img src="count.svg"></li>
+                                <li class="details">Back End Dev</li>
+                                <li class="details"><img src="count.svg"></li>
+                                <li class="details">2023</li>
+                            </ul>
+                        </div>
+                        <p class="infos">A daily selection of privately personalized reads; no accounts
+                            or sign-ups required.
+                        </p>
+                        <ul class="languagess">
+                            <li class="languages" id="html1">HTML</li>
+                            <li class="languages" id="css1">CSS</li>
+                            <li class="languages" id="javascript1">Javascript</li>
                         </ul>
+                        <button class="buttons">see project</button>
```

Comment:
- [optional] Great job fixing the mobile version as requested by previous reviewer. Please kindly try to implement the hover and pressed effect on all the buttons on the desktop version. This will ensure your design perfectly aligns with the Figma version.
![Screenshot (1112)](https://github.com/Omwomo/Microverse-web-portfolio-project/assets/65298048/37d36ca2-6957-4486-a683-cd8f949bf297)